README.md

```diff
@@ -32,7 +32,8 @@ Documentation: https://mmtracking.readthedocs.io/
 
 ## Introduction
 
-MMTracking is an open source video perception toolbox by PyTorch. It is a part of [OpenMMLab](https://openmmlab.com) project.
+MMTracking is an open source video perception toolbox based on PyTorch.
+It is a part of the OpenMMLab project.
```

Comment:
revert the changes

Reply:
Oh, this was a mistake. I have reverted it in the new commit.